02-esc/modal.js

```diff
@@ -15,27 +16,38 @@ export function openModal(e) {
 	idVariabelForModal = parseInt(e.target.dataset.id);
 	minParticipantsVaribelForModal = e.target.dataset.minparticipants;
 	maxParticipantsVaribaleForModal = e.target.dataset.maxparticipants;
+	roomTitleforModal = e.target.dataset.title;
 	let body = document.querySelector("body");
 	modalDiv = document.createElement("div");
 	modalDiv.className = "modal-div";
 	modalDiv.innerHTML = `
+		<div class="modalContents">
```

Comment:
There is some weird indentation going on in this code. Reading HTML strings in javascript is always difficult, but it's made more difficult by the inconsistent indentation.